django/forms/models.py

```diff
@@ -1239,6 +1240,8 @@ def prepare_value(self, value):
         return super().prepare_value(value)
 
     def to_python(self, value):
+        if isinstance(value, models.Model):
```

Comment:
Should this be `isinstance(value, self.queryset.model)`?

Reply:
code changed